custom_components/samsungtv_tizen/media_player.py

```diff
@@ -445,6 +513,9 @@ def source_list(self):
         if self._app_list is None:
             self._gen_installed_app_list()
 
+        if self._power_off_in_progress() or self._state == STATE_OFF:
+            return None
```

Comment:
Is there any benefit of returning `None` for `source_list()`? `source_list` is just an array of options for changing the input source that is used for the HA user interface, and if `state()` returns `OFF`, then the option of changing the input source isn't shown at all in the HA user interface. But maybe I'm missing something that I don't know of..

Reply:
Yes, because some lovelace alternative media interface (as the very good "mini media player") that always return the source_list() also if the state is OFF. In this way if the status is OFF the list is always not shown, that is the correct thing.

Reply:
Ok, if you say there's a benefit to this, then I'll accept it.